app/models/finance/statement/ecf/paid.rb

```diff
@@ -4,4 +4,8 @@ class Finance::Statement::ECF::Paid < Finance::Statement::ECF
   def open?
     false
   end
+
+  def paid?
```

Comment:
is this correct, do we trust that all  records have the type` Finance::Statement::ECF::Paid` when they are paid, or is this calculated via the output fee/payment and deadline date?

Reply:
@cwrw the `Finance::Statement::ECF::Paid` are manually flipped to paid. So you should be able to trust that all the statement line items should be either `paid`, `clawed_back` or `voided`. Although I doubt this has any kind of  validation as it was done hastily.